File: Image2Text/Image2Text/app/src/main/java/com/rahulsoni0/image2text/MainActivity.java
package com.rahulsoni0.image2text;

import androidx.appcompat.app.AppCompatActivity;

import android.app.ActionBar;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.view.WindowManager;

import com.rahulsoni0.image2text.databinding.ActivityMainBinding;
import com.rahulsoni0.image2text.tools.ScannerActivity;

public class MainActivity extends AppCompatActivity {
    private ActivityMainBinding binding;// viewBinding

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        if (getSupportActionBar() != null) {
            getSupportActionBar().hide();
        }

        binding.btnCapture.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent i = new Intent(MainActivity.this, ScannerActivity.class);
                startActivity(i);
            }
        });


    }
}

File: GreetingCard App/GreetingCard/app/src/main/java/com/akshatbhuhagal/greetingcard/MainActivity.kt
package com.akshatbhuhagal.greetingcard

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import com.akshatbhuhagal.greetingcard.databinding.ActivityMainBinding

const val EXTRA_MESSAGE = "com.akshatbhuhagal.greetingcard.MESSAGE"

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        supportActionBar!!.hide()



            binding.btnWelcome.setOnClickListener {

                val userName = binding.etName.text.toString()

                val intent = Intent(this, WelcomeActivity::class.java).apply {

                    putExtra(EXTRA_MESSAGE, userName)

                }

                if (binding.etName.text.isNullOrEmpty()) {
                    Toast.makeText(this, "Enter your Name", Toast.LENGTH_SHORT).show()
                } else {
                    startActivity(intent)
                }


            }


    }
}